db/models/review.js

```diff
@@ -15,8 +15,8 @@ module.exports = db => db.define('reviews', {
 })
 
```

Comment:
Think we should give it more than 30 characters in length -- that's not even a Tweet!

I think the ARRAY(TEXT) structure we saw in Auther was clever.